rebar.config

```diff
@@ -40,7 +41,7 @@
         {h3, ".*", {git, "https://github.com/helium/erlang-h3.git", {branch, "master"}}},
         {ubx, ".*", {git, "https://github.com/helium/erlang-ubx", {branch, "master"}}},
         {gatt, ".*", {git, "https://github.com/helium/ebus-gatt", {branch, "master"}}},
-        {connman, ".*", {git, "https://github.com/helium/ebus-connman", {branch, "master"}}},
+        {connman, ".*", {git, "https://github.com/helium/ebus-connman", {branch, "remove-wifi-service"}}},
```

Comment:
We need ebus-connman to merge so we can fix this up, right?

Reply:
That ebus-connman  feature is now merger: https://github.com/helium/ebus-connman/pull/4 Thanks @fvasquez